build/index.html

```diff
@@ -321,8 +321,8 @@ <h1>From built file remove code:</h1>
         ifSpecifiedInclude('itext', 'src/mixins/itext_key_behavior.mixin.js'),
         ifSpecifiedInclude('itext', 'src/mixins/itext.svg_export.js'),
 
-        ifSpecifiedInclude('itext', 'src/shapes/textbox.class.js'),
-        ifSpecifiedInclude('itext', 'src/mixins/textbox_behavior.mixin.js'),
+        ifSpecifiedInclude('textbox', 'src/shapes/textbox.class.js'),	
```

Comment:
do we have a textbox id on the page? i do not see it.

Reply:
My bad ! Fixed